libr/asm/arch/avr/assemble.c

```diff
@@ -0,0 +1,360 @@
+#include "assemble.h"
+#include "r_asm.h"
+#include "format.h"
+
+
+extern instructionInfo instructionSet[AVR_TOTAL_INSTRUCTIONS];
+
```

Comment:
why so empty

Reply:
because the latest commit was not finished yet. You can ignore for this moment.